client/verta/tests/registry/pydantic_models.py

```diff
@@ -0,0 +1,51 @@
+from dataclasses import dataclass
+
+from pydantic import BaseModel
+
+
+@dataclass
+class AnInnerClass(BaseModel):
+    h_dict: dict
+    i_list_str: list[str]
+
+    def __init__(self, h, i):
+        super().__init__(
+            h_dict=h,
+            i_list_str=i,
+        )
+
+
+@dataclass
+class AClass(BaseModel):
```

Comment:
Based on the way we're using these, I think we should give them more descriptive names like

```suggestion
class InputModel(BaseModel):
```

and `OutputModel`. Same with the `generate_*()` composite strategies.

Or at least, I'm doubtful these would get re-used for anything else, so might as well name them according to what they're doing for now